tests/filesystem.go

```diff
@@ -156,7 +156,9 @@ func runIgnition(t *testing.T, ctx context.Context, stage, root, cwd string, app
 		"-root", root, "-log-to-stdout", "--config-cache", filepath.Join(cwd, "ignition.json")}
 	cmd := exec.CommandContext(ctx, "ignition", args...)
 	t.Log("ignition", args)
+	// nolint:staticcheck
```

Comment:
Let's keep the explicit nil check, and then add a comment here saying that the linter warns anyway.